xrayvision/Visibillity.py

```diff
@@ -215,3 +215,56 @@ def idft_map(input_visibilities, output_map, input_uv):
                     2j * np.pi * (input_uv[0, :] * x[i] + input_uv[1, :] * y[i])))
 
         return im.reshape(m, n)
+
+
+class RHESSIVisibility(Visibility):
+    """
+    A set of RHESSI visibilities.
+
+    Parameters
+    ----------
+    uv: `numpy.ndarray` The u, v coordinates of the visibilities
+    vis: `numpy.ndarray` The complex visibility
+    isc: `int` Related to the grid/detector
+    harm: `int` Harmonic used
+    erange: `numpy.ndarray` Energy range
+    trange: `numpy.ndarray` Time range
+    totflux: `float` Total flux
+    sigamp: `float` Sigma or error on visibility
+    chi2: `float` Chi squared from fit
+    xyoffset: `np.ndarray` Offset from Sun centre
+    type_string: `str` count, photon, electron
+    units: `str`
+    atten_state: `int` State of the attenuator
+    count: `float` detector counts
+    Examples
+    --------
+
+    Notes
+    -----
+
+    """
+
+    def __init__(self, uv, vis, isc: int=0, harm: int=1,
+                 erange: np.array=np.array([0.0, 0.0]),
+                 trange: np.array=np.array([0.0, 0.0]),
+                 totflux: float=0.0, sigamp: float=0.0,
+                 chi2: float=0.0,
+                 xyoffset: np.array=np.array([0.0, 0.0]),
+                 type_string: str="photon",
+                 units: str="Photons cm!u-2!n s!u-1!n",
```

Comment:
This is IDL syntax to get super and sub scripts (!u for upper and !l for lower) the equivalent latex which matplotlib can display would be ```"cm^{-2}` s^{-1}"```